packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/.eslintrc.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
this is a case for multi clients, we need to generate the serialize/deserialize utils for each client separately.

Reply:
I would say, the client is divided conceptually, normally it's because architects think some of the operations should be grouped into a relatively independent client. it's very likely we will have models being used in more than one client. Since we are generating them in each separate subclient folder and we will not export the serialize util,  I am not too worried about the serializer implementation, just a small reminder of the file path will be different.